src/Symfony/Component/HttpKernel/Attribute/Cache.php

```diff
@@ -50,7 +54,12 @@ public function __construct(
         public bool $mustRevalidate = false,
 
         /**
-         * Additional "Vary:"-headers.
+         * Set "Vary" header.
+         *
+         * Example:
+         * #[Cache(vary: ['Accept-Encoding', 'User-Agent'])]
```

Comment:
```suggestion
         * ['Accept-Encoding', 'User-Agent']
```
I propose to just use the value here

Reply:
Yes, I think that's easier to understand here.